cmd/sql/flow_test.go

```diff
@@ -170,6 +170,15 @@ func TestFlowRunCmd(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestDebugFlowRunCmd(t *testing.T) {
+	projectDir := t.TempDir()
+	err := execFlowCmd([]string{"init", projectDir}...)
+	assert.NoError(t, err)
+
+	err = execFlowCmd([]string{"--debug", "run", "example_templating", "--env", "dev", "--project-dir", projectDir}...)
+	assert.NoError(t, err)
```

Comment:
Is checking if the command didn't raise an exception good enough? Would there be any other meaningful assertions we could make here?

Reply:
I think that this is currently enough as cobra will raise an error if the command is incorrect e.g. has too many args or unknown flags, etc.